core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-results-seo-tool/dot-results-seo-tool.component.scss

```diff
@@ -0,0 +1,156 @@
+
+.results-seo-tool__main-card-content {
+    display: flex;
+    flex-direction: column;
+}
+
+.results-seo-tool__main-card-title {
+    color: $black;
+    font-size: $font-size-sm;
+    font-style: normal;
+    font-weight: 700;
+    line-height: normal;
+    margin: $spacing-3 0;
+    display: block;
+}
+
+.results-seo-tool__card-title {
+    display: flex;
+    align-items: center;
+    color: $black;
+    gap: $spacing-3;
+    font-size: $font-size-md;
+    margin-bottom: $spacing-3;
+}
+
+.results-seo-tool__card {
+    border-radius: $border-radius-lg;
+    border: 1px solid $color-palette-gray-300;
+    padding: $spacing-3;
+}
+
+.results-seo-tool__card-favicon {
+    border-radius: $card-favicon-radius;
+    background: $color-palette-gray-200;
+    padding: $spacing-1;
+}
+
+.results-seo-tool__search-title {
+    font-size: $font-size-lg;
+    text-decoration: none;
+}
+
+.results-seo-tool__search-description {
+    color: $black;
+    font-size: $spacing-3;
+}
+
+.results-seo-tool__version--small {
+    width: $version-small;
+}
+
+.results-seo-tool__result-card {
+    width: $result-card-width;
+    margin-bottom: $spacing-4;
+}
+
+.results-seo-tool__result-card-header {
+    border-bottom: 1px solid $color-palette-gray-400;
+}
+
+.results-seo-tool__result-card-title {
+    display: flex;
+    gap: $spacing-3;
+    align-items: center;
+    font-size: $font-size-lg;
+    color: $black;
+}
+
+.results-seo-tool__result-card-list {
+    list-style: none;
+    padding: $spacing-4 $spacing-7;
+    margin: 0;
+}
+
+.results-seo-tool__result-card-list li {
+    display: flex;
+    gap: $spacing-3;
+    color: $black;
+}
+
+.results-seo-tool__result-card-list li:not(:last-child) {
+    margin-bottom: $spacing-3;
+}
+
+.results-seo-tool__result-card-info {
+    width: 100%;
+    background: $color-palette-primary-100;
+    padding: $spacing-1 $spacing-3;
+    border-bottom-left-radius: $border-radius-xl;
+    border-bottom-right-radius: $border-radius-xl;
+    display: flex;
+    align-items: flex-start;
+    gap: $spacing-1;
+    line-height: $line-height;
+    color: $color-palette-gray-900;
+    font-size: $font-size-sm;
+}
+
+.results-seo-tool__result-icon {
+    font-size: $font-size-xl;
+    color: $color-palette-primary-500;
+}
+
+::ng-deep {
+    .results-seo-tool__result-card .p-card {
+        border: 1px solid $color-palette-gray-400;
+        border-radius: $border-radius-xl;
+    }
+
+    .results-seo-tool__result-card .p-card .p-card-header {
+        border-bottom: 1px solid $color-palette-gray-400;
+        padding: $spacing-3;
+    }
+
+    .results-seo-tool__result-card .p-card .p-card-body {
+        padding: 0;
+    }
+
+    .results-seo-tool__main-card .p-card {
+        border-radius: $border-radius-xl;
+    }
+
+    .results-seo-tool__result-card .p-card-content {
+        flex-direction: column;
+    }
+
+    .results-seo-tool__result-card-list .pi-external-link {
+        color: $color-palette-primary-500;
+    }
+}
```

Comment:
While working in the global styling @zJaaal point something important and is that is safer to put the `::ng-deep` inside the `:host` to make sure this will not leak out of this component, even tho you already have `results-seo-tool__result-card`, `:host` is an extra layer.

Reply:
Move the `:host ::ng-deep`